backend/src/lib.rs

```diff
@@ -80,6 +80,7 @@ pub fn setup_actix_app(
         .service(scope("/event").configure(routes::event::config))
         .service(scope("/flag").configure(routes::flag::config))
         .service(scope("/leaderboard").configure(routes::leaderboard::config))
+        .route("/sse", actix_web::web::get().to(services::sse_handler::sse_handler))
```

Comment:
I don't like this name. Just name it something like `/events`. You can probably event move it to `flags/` endpoint